buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -785,6 +836,10 @@ private void startOpenSearchProcess() {
         environment.clear();
         environment.putAll(getOpenSearchEnvironment());
 
+        if (!extensions.isEmpty()) {
+            environment.put("OPENSEARCH_JAVA_OPTS", "-Dopensearch.experimental.feature.extensions.enabled=true");
```

Comment:
For my understanding. Why do we need to enable the flag again if we are adding the experimental flag [here](https://github.com/opensearch-project/OpenSearch/pull/7235/files#diff-0eecf581460b128c3d70a559dfdabece54a006048cc0a4e773d020ce1f04fc9bR382)?

Reply:
I forgot to remove the other one.  This is where we are actually setting the feature flag.